cmd/agent/app/integrations.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	tufCmd.AddCommand(searchCmd)
 	tufCmd.AddCommand(freezeCmd)
 	tufCmd.PersistentFlags().BoolVarP(&withoutTuf, "no-tuf", "t", false, "don't use TUF repo")
+	tufCmd.PersistentFlags().BoolVarP(&inToto, "phase-1", "1", false, "enable in-toto")
```

Comment:
is this supposed to be used by customers ? If yes, let's use a more friendly name

Reply:
Yes it is for beta testing. I just used @trishankatdatadog naming. Will see with him if we want another name.

Reply:
Let's use `--in-toto` instead of `--phase-1`: that way, it's more consistent with `--no-tuf`, too. Please note: if `--no-tuf` is set, then `--in-toto` must be rejected.